src/common/datetime/format_date.ts

```diff
@@ -7,10 +7,12 @@ if (__BUILD__ === "latest" && polyfillsLoaded) {
 }
 
 // Tuesday, August 10
-export const formatDateWeekday = (dateObj: Date, locale: FrontendLocaleData) =>
-  formatDateWeekdayMem(locale).format(dateObj);
+export const formatDateWeekdayDay = (
```

Comment:
Was not really named correctly, since it does not only return the weekday, but also the date => renamed to make room for the pure weekday version further down below.